src/main/kotlin/com/wafflestudio/toyproject/team4/core/user/service/AuthService.kt

```diff
@@ -12,11 +13,13 @@ interface AuthService {
 
 @Service
 class AuthServiceImpl(
+    private val authConfig: AuthConfig,
```

Comment:
`AuthConfig`보다 `PasswordEncoder`를 바로 의존성 주입해서 사용하면 코드도 간결해지고 더 좋을 것 같습니다.
```suggestion
    private val passwordEncoder: PasswordEncoder
```